torch_geometric/utils/spmm.py

```diff
@@ -65,7 +65,11 @@ def spmm(src: Adj, other: Tensor, reduce: str = "sum") -> Tensor:
 
     if reduce == 'sum':
         return torch.sparse.mm(src, other)
+    elif reduce == 'mean':
+        src = src.coalesce()
+        deg = degree(src.indices()[0], num_nodes=src.size(0))
```

Comment:
Can we add a TODO here to utilize `rowptr` in case `src` is given as a CSR matrix?

Reply:
Done.